src/Components/TrainingListPage.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Text, ImageBackground, TouchableOpacity, FlatList } from 'react-native';
+import { connect } from 'react-redux';
+import theme from '../styles/theme.style';
+import * as list from '../styles/list.style';
+
+const getTrainingDuration = trainingDrills => {
+  const durationList = trainingDrills.map(({ durationInMinutes }) => durationInMinutes);
+  return durationList.reduce((a, b) => a + b, 0);
+};
+
+const getTrainingNbPlayers = trainingDrills => {
+  const nbPlayersList = trainingDrills.map(({ nbPlayers }) => nbPlayers);
+  return Math.max(...nbPlayersList);
+};
+
+const mapStateToProps = state => {
+  return {
+    trainings: state.trainings,
+    allDrills: state.drills,
+  };
+};
+
+export const TrainingListPage = props => {
+  const { allDrills, trainings, navigation } = props;
+
+  const [data] = useState(trainings);
+
+  return (
+    <View style={styles.trainingListPage}>
+      <Text style={list.counter}>{data.length} training sessions available</Text>
+      <FlatList
+        data={data}
+        keyExtractor={item => item.id.toString()}
+        renderItem={({ item }) => (
+          <TouchableOpacity style={list.item} onPress={() => navigation.navigate('TrainingPage', { training: item })}>
```

Comment:
I guess you could add a test on navigation (there should be an example now in DrillListPage.test (the debug variable is probably useless there btw)

Reply:
done

Reply:
Thanks